.github/workflows/ci.yml

```diff
@@ -46,3 +47,14 @@ jobs:
 
       - name: Run tests
         run: npm test
+
+      - name: Run coverage
```

Comment:
I'm not quite sure if this part will ever run if line 49 (`run: npm test`) failed
We could try with a forced-failing test

Reply:
If the test fails then coverage should normally not run.